scripts/install/EnableUuidQtiIdentifier.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2024 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\scripts\install;
+
+use common_ext_ExtensionsManager;
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\featureFlag\FeatureFlagChecker;
+use oat\tao\model\featureFlag\FeatureFlagCheckerInterface;
+
+class EnableUuidQtiIdentifier extends InstallAction
+{
+    public function __invoke($params): void
+    {
+        if ($this->customIdentifierGenerationStrategyEnabled()) {
+            $this->addQtiCreatorConfig();
+            $this->addClientLibRegistryEntry();
+        }
+    }
+
+    private function addClientLibRegistryEntry(): void
+    {
+        $taoExtension = $this->getServiceManager()
+            ->get(common_ext_ExtensionsManager::SERVICE_ID)
+            ->getExtensionById('tao');
+
+        $config = $taoExtension->getConfig('client_lib_config_registry');
+
+        if (!isset($config['taoQtiItem/qtiCreator/widgets/helpers/qtiIdentifier'])) {
+            $config['taoQtiItem/qtiCreator/widgets/helpers/qtiIdentifier'] = [
+                'qtiIdPattern' => '/^\d{9}$/'
+            ];
+        }
+
+        $taoExtension->setConfig('client_lib_config_registry', $config);
+    }
+
+    private function addQtiCreatorConfig(): void
+    {
+        $taoQtiItemExtension = $this->getServiceManager()
+            ->get(common_ext_ExtensionsManager::SERVICE_ID)
+            ->getExtensionById('taoQtiItem');
+        $config = $taoQtiItemExtension
+            ->getConfig('qtiCreator');
+
+        if (!isset($config['identifierGenerationStrategy'])) {
+            $config['identifierGenerationStrategy'] = 'uuid';
```

Comment:
I believe it will be better to rename the strategy from `uuid` to something else e.g. `unique-numeric`?
It might create expectation to have  identifier like `[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}` as it already well known in industry under this acronym as well as what you have in top of google search about it 
<img width="1354" alt="image" src="https://github.com/oat-sa/extension-tao-itemqti/assets/37804293/ccbd9435-eb1f-4bc8-a4e7-e8465f3d815c">

Reply:
I have same impression, lets rename it to something else, also I suggested we can use Snowflake ID which guarantees almost no ID collisions https://en.wikipedia.org/wiki/Snowflake_ID

Reply:
Agree with a renaming this strategy to unique-numeric. 

@Karol-Stelmaczonek regarding Snowflake ID my approach is very similar to presented in this standard. I could not apply fully snowflake standard cause I need to adjust to requirement from client. Therefore I choose to take 7 digits from timestamp and 2 digits from random value generated by individual browser. Unless I misunderstand your suggestion.